website/content/partials/known-issues/config_listener_proxy_protocol_behavior_issue.mdx

```diff
@@ -0,0 +1,33 @@
+<a id="listener-proxy-protocol-config" />
+
+### Potential DoS when using the `deny_unauthorized` proxy protocol behavior for a TCP listener
+
+#### Affected versions
+
+##### Community Edition (CE)
+
+- 1.10.x - 1.15.x
+- 1.17.1
+
+##### Enterprise
+
+- 1.10.x+ent - 1.15.11+ent
+- 1.16.5+ent
+- 1.17.1+ent
+
+#### Issue
+
+Vault TCP listeners configured with the `deny_unauthorized` [proxy_protocol_behavior](/vault/docs/configuration/listener/tcp#proxy_protocol_behavior)
+which receive a request from an untrusted upstream connection will close, causing
+Vault to no longer respond to requests via that listener.
+
+This leaves the Vault API vulnerable to a denial-of-service (DoS) attack by requests
+which are intentionally made by untrusted upstream connections, causing it to become
+unavailable on that node.
+
+This issue is addressed in Vault 1.15.12+ent, 1.16.6+ent, 1.17.2, 1.17.2+ent and later.
```

Comment:
What is "this"? The fact that the listener closes or that it leaves Vault susceptible to DoS attacks?

Reply:
Both,  by stopping the listener from closing we stop the ability for the DoS to be exploited in this way.